app/models/stats_admin.rb

```diff
@@ -0,0 +1,8 @@
+# -*- coding: utf-8 -*-
+class StatsAdmin
```

Comment:
@mjgiarlo I'm not sure why we are generating it into the app.  For easier overriding?  With it being defined we do not need to check the route to see if it exists, so the logic is simpler.